api/src/main/java/edu/cornell/mannlib/vitro/webapp/dynapi/ResourceAPIPool.java

```diff
@@ -0,0 +1,30 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi;
```

Comment:
Note, should attempt to use git mv instead of deleting and creating new file. Some IDEs do this correctly for you, some don't.

Reply:
excellent point. I have used IDE (IntelliJ) to rename file (and to update other files referencing that class). The problematic part is that the history of the file is lost. I have tried to resolve this, and not sure it is correct now. I have noticed that renaming of ResourceKey to ResourceAPIKey is well displayed in the GitHub after my new commit, while those classes where you commented not.

Reply:
`ResourceAPIPool` is still a new file and not renamed.